### PR TITLE
fix notes config-scanning URL

### PR DESCRIPTION
**Pull request by Bezbran**
## Describe your changes
fix notes config-scanning URL
## Screenshots - If Any (Optional)
image

### CODE REVIEW

File: .github/workflows/03-helm-release.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+          
+      - name: git checkout to new commit
+        run: git checkout ${{ inputs.COMMIT_REF }}
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+        # with:
+        #   version: v3.4.0
```
Comment (dwertent) [inline .github/workflows/03-helm-release.yaml:43]: @matanshk Why did you remove this?

File: .github/workflows/03-helm-release.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+name: 03-Helm chart release
+
+on:
```
Comment (dwertent) [inline .github/workflows/03-helm-release.yaml:3]: We should ignore paths:
```
    paths:
      - 'charts/**.yaml'
      - 'charts/**.tpl'
```
cc: @matanshk

Comment (matanshk) [reply .github/workflows/03-helm-release.yaml:3]: @dwertent 
please talk with me, I want to understand why

Comment (dwertent) [reply .github/workflows/03-helm-release.yaml:3]: Because we do not want the workflow to run when changes are made to these paths

Comment (matanshk) [reply .github/workflows/03-helm-release.yaml:3]: The workflow will be triggered only by the in-cluster component or by triggering it manually, it does not have any other triggers (like push); so it will never run if we are changing something in this paths anyway

File: .github/workflows/03-helm-release.yaml (hunk 1 above)
Comment (dwertent) [reply .github/workflows/03-helm-release.yaml:43]: yes

File: .github/workflows/03-helm-release.yaml (hunk 2 above)
Comment (dwertent) [reply .github/workflows/03-helm-release.yaml:3]: We cannot trigger this workflow when updating the helm charts manually? What if I add a field?

**Review by dwertent — APPROVED** [review]
